packages/chan-ko-website/src/components/AboutUs/AboutUs.tsx

```diff
@@ -19,35 +19,6 @@ const AboutUs: React.FC = () => {
             help clients navigate the rapidly evolving digital landscape.
           </p>
         </div>
-        <div className={styles.highlights}>
-          <div className={styles.highlight}>
-            <h3>Technology Expertise</h3>
-            <ul>
-              <li>Fractional CTO Services</li>
-              <li>Cloud Computing & Big Data</li>
-              <li>Artificial Intelligence & Machine Learning</li>
-              <li>Blockchain & Cryptocurrency</li>
-            </ul>
-          </div>
-          <div className={styles.highlight}>
-            <h3>Entertainment Focus</h3>
-            <ul>
-              <li>Digital Content Creation</li>
-              <li>Streaming Platforms</li>
-              <li>Interactive Media</li>
-              <li>Gaming & eSports</li>
-            </ul>
-          </div>
-          <div className={styles.highlight}>
```

Comment:
Removing these lines seems to remove any usage of the CSS classes `highlights` & `highlight`, so we should be removing any related classes from the `Hero.module.css` file